pkg/basicserver/metrics.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 TiKV Project Authors.
```

Comment:
We don't need this package in this version.

Reply:
done